src/core/simulation_results.jl

```diff
@@ -638,7 +638,7 @@ end
 """
 Construct SimulationResults from a simulation.
 """
-SimulationResults(sim::Simulation) = SimulationResults(get_simulation_dir(sim))
+SimulationResults(sim::Simulation; kwargs...) = SimulationResults(get_simulation_dir(sim); kwargs...)
```

Comment:
I prefer to have the kwargs explicitly to that the function fails if an invalid option is passed

Reply:
IMHO, when you're simply forwarding kwargs to another function, what Clayton did is better. He is explicitly listed the arguments in the function that consumes them.